gensim/models/fasttext.py

```diff
@@ -1282,3 +1287,40 @@ def _check_model(m):
             "mismatch between final vocab size (%s words), and expected vocab size (%s words)",
             len(m.wv.vocab), m.vocabulary.vocab_size
         )
+
+
+def save_facebook_model(model, path, encoding="utf-8", lr_update_rate=100, word_ngrams=1):
+    """Saves word embeddings to the Facebook's native fasttext `.bin` format.
+
+    Notes
+    ------
+    Facebook provides both `.vec` and `.bin` files with their modules.
+    The former contains human-readable vectors.
+    The latter contains machine-readable vectors along with other model parameters.
+    This function saves only the .bin file**.
+
+    Parameters
+    ----------
+    model : gensim.models.fasttext.FastText
+        FastText model to be saved.
+    path : str
+        Output path and filename (including `.bin` extension)
+    encoding : str, optional
+        Specifies the file encoding. Defaults to utf-8.
+
+    lr_update_rate : int
+        This parameter is used by Facebook faasttext tool, unused by Gensim.
```

Comment:
Typo.

```suggestion
        This parameter is used by Facebook fasttext tool, unused by Gensim.
```

Please check elsewhere, too.

Reply:
Corrected here and in one more place.